### DISCUSSION TITLE
About Amazon Partner Network

### DISCUSSION THREAD

**Discussion by alpapazzo** [amzn/selling-partner-api-models#3122] [General] [answered]
Hello, 
I read there is the new site 'Amazon Partner Network'. 
But how can I list a SP API App in Amazon Partner Network ? Are there some requirements ?

**Comment by jlevers** [comment, marked as answer]
You have to publish your app to get it listed it on the Amazon Partner Network. To start publishing an app, click the dropdown next to it on the right side of the Seller Central `Apps & Services > Develop Apps` page, and select `Edit listing`.

For users to be able to authorize your app after it's published, you need to set up the APN authorization flow ([documentation here](https://github.com/amzn/selling-partner-api-docs/blob/main/guides/en-US/developer-guide/SellingPartnerApiDeveloperGuide.md#amazon-partner-network-authorization-workflow)).

**Comment by alpapazzo** [comment]
Thank you very much I didn't see the Edit Listing link, thanks.
